templates/Includes/SimpleStyleguideNavigation.ss

```diff
@@ -0,0 +1,12 @@
+<div class="styleguide__navigation-wrap">
+  <h3>Quick links</h3>
+  <ul class="styleguide__navigation">
+    <%--<li class="styleguide__navigation-item"><a href="#colors">Colors</a></li>--%>
```

Comment:
The style guide module supports the ability to predefine some colour combination to display in the guide.

That might not be practical in our set up because the colour can be define dynamically.